src/utils/components/CloneTemplateModal/CloneTemplateModal.tsx

```diff
@@ -64,6 +65,8 @@ const CloneTemplateModal: React.FC<CloneTemplateModalProps> = ({
           ...draftTemplate?.metadata?.labels,
           'template.kubevirt.io/type': TEMPLATE_TYPE_VM,
```

Comment:
i know its not related to your PR, but maybe change this also to a const? add to LABELS?